sources/packages/web/src/components/aest/AssessApplicationOfferingChangeRequestModal.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <v-form ref="assessApplicationOfferingChangeRequestForm">
+    <modal-dialog-base
+      :showDialog="showDialog"
+      :title="getTitle"
+      :max-width="780"
+    >
+      <template #content
+        ><error-summary
+          :errors="assessApplicationOfferingChangeRequestForm.errors"
+        />
+        <p class="mt-4">
```

Comment:
just want to double-check, do we really need these margins and padding with `p` tags? Lines 12, 23, 32

Reply:
The problem is that the default vertical margin of the `v-divider` is too high. I changed to no margin in the common component and gave the margin as required in this component to the elements instead.

Reply:
Updated this component as well with the style changes.